### ISSUE TITLE
Cannot run laravel tinker with microscope installed

### ISSUE THREAD

**Issue by jakobbuis** [imanghafoori1/laravel-microscope#137]
**Steps to reproduce**
1. Start a laravel application
2. install laravel-microscope v1.0.167
3. Run `php artisan tinker`

**Expected:** laravel tinker started
**Happened:** error, and tinker crashed
```
php artisan tinker                              

   Error 

  Token T_NAME_FULLY_QUALIFIED has same ID as token T_NAME_QUALIFIED, you may be using a library with broken token emulation

  at vendor/nikic/php-parser/lib/PhpParser/Lexer.php:438
    434▕             if (\defined($token)) {
    435▕                 $tokenId = \constant($token);
    436▕                 $clashingToken = $usedTokenIds[$tokenId] ?? null;
    437▕                 if ($clashingToken !== null) {
  ➜ 438▕                     throw new \Error(sprintf(
    439▕                         'Token %s has same ID as token %s, ' .
    440▕                         'you may be using a library with broken token emulation',
    441▕                         $token, $clashingToken
    442▕                     ));

      +21 vendor frames 
  22  artisan:35
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
```

**Comment by jakobbuis** [comment]
This bug also occurs when installing dev-master, currently at `6395006`

**Comment by lcandellier** [comment]
I have the same problem with versions 1.0.166 and 1.0.167 and as environment php 7.4 and laravel 8.42

I fixed by downgrading to 1.0.165.

**Comment by imanghafoori1** [comment]
please try the latest patch. it should work

**Comment by jakobbuis** [comment]
Can confirm that it is fixed in `a31bed0cc795672269e21393f69e6d144b687030`

**Comment by lcandellier** [comment]
Fixed also on my side, thank you for the quality of your package !
